### PR TITLE
Nerfs Krav Maga

### PR DESCRIPTION
**Pull request by Arzion5**
:cl:
tweak: Krav Maga has been nerfed. The knockdown is shorter and you suffocate less from lung punches.
/:cl:
Modularized and hopefully works.

### CODE REVIEW

File: hippiestation/code/datums/martial/krav_maga.dm [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/datum/martial_art/krav_maga/leg_sweep(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)
+	if(D.stat || D.IsKnockdown())
+		return 0
+	D.visible_message("<span class='warning'>[A] leg sweeps [D]!</span>", \
+					  	"<span class='userdanger'>[A] leg sweeps you!</span>")
+	playsound(get_turf(A), 'sound/effects/hit_kick.ogg', 50, 1, -1)
+	D.apply_damage(5, BRUTE)
+	D.Knockdown(20)
+	add_logs(A, D, "leg sweeped")
+return 1
```
Comment (KayeArray) [inline hippiestation/code/datums/martial/krav_maga.dm]: return TRUE

File: hippiestation/code/datums/martial/krav_maga.dm [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/datum/martial_art/krav_maga/leg_sweep(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)
+	if(D.stat || D.IsKnockdown())
+		return 0
```
Comment (KayeArray) [inline hippiestation/code/datums/martial/krav_maga.dm]: return FALSE

**Comment by ghost** [comment]
Top 10 annoying PRs that will have you screaming at the monitor for 3 hours.

File: hippiestation/code/datums/martial/krav_maga.dm [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+	D.apply_damage(5, BRUTE)
+	D.Knockdown(20)
+	add_logs(A, D, "leg sweeped")
+return 1
+
+/datum/martial_art/krav_maga/quick_choke(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)//is actually lung punch
+	D.visible_message("<span class='warning'>[A] pounds [D] on the chest!</span>", \
+				  	"<span class='userdanger'>[A] slams your chest! You can't breathe!</span>")
+	playsound(get_turf(A), 'sound/effects/hit_punch.ogg', 50, 1, -1)
+	if(D.losebreath <= 10)
```
Comment (KayeArray) [inline hippiestation/code/datums/martial/krav_maga.dm]: this if isn't needed

File: hippiestation/code/modules/uplink/uplink_item.dm [hunk 4]
```diff
@@ -233,3 +233,10 @@
 	item = /obj/item/storage/box/syndie_kit/football
 	cost = 16
 	exclude_modes = list(/datum/game_mode/nuclear)
+	
+/datum/uplink_item/badass/horsecock
```
Comment (KayeArray) [inline hippiestation/code/modules/uplink/uplink_item.dm]: controversial

**Comment by ghost** [comment]
fine with krab magooba change honestly since it's pretty fukken strong but i'd rather no have the dildos buyable as tator tot

File: hippiestation/code/datums/martial/krav_maga.dm [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+		D.losebreath = Clamp(D.losebreath + 2, 0, 10)
+	D.adjustOxyLoss(10)
+	add_logs(A, D, "quickchoked")
+	return 1
```
Comment (KayeArray) [inline hippiestation/code/datums/martial/krav_maga.dm]: return TRUE

**Review by KayeArray — APPROVED** [review]
a few nitpicks

**Comment by Arzion5** [comment]
Enough support and dildos will be delet

**Comment by GuyonBroadway** [comment]
really the dildos thing should be a separate PR.

**Comment by DinduStuffin** [comment]
Can you make the message for stomping people with Krav Maga a bit bigger? I think it might cut down on situations like IvanTheCracka's latest ban.

Make the dildo stuff another PR and I'll +1 this.

File: hippiestation/code/datums/martial/krav_maga.dm [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+	D.apply_damage(5, BRUTE)
+	D.Knockdown(20)
+	add_logs(A, D, "leg sweeped")
+return TRUE
+
+/datum/martial_art/krav_maga/quick_choke(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)//is actually lung punch
+	D.visible_message("<span class='warning'>[A] pounds [D] on the chest!</span>", \
+				  	"<span class='userdanger'>[A] slams your chest! You can't breathe!</span>")
+	playsound(get_turf(A), 'sound/effects/hit_punch.ogg', 50, 1, -1)
+	(D.losebreath <= 10)
```
Comment (KayeArray) [inline hippiestation/code/datums/martial/krav_maga.dm]: i meant remove the whole if statement not just the word 'if'

Comment (KayeArray) [reply hippiestation/code/datums/martial/krav_maga.dm]: make sure you un indent the block below as well

File: hippiestation/code/datums/martial/krav_maga.dm [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+	D.apply_damage(5, BRUTE)
+	D.Knockdown(20)
+	add_logs(A, D, "leg sweeped")
+return TRUE
+
+/datum/martial_art/krav_maga/quick_choke(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)//is actually lung punch
+	D.visible_message("<span class='warning'>[A] pounds [D] on the chest!</span>", \
+				  	"<span class='userdanger'>[A] slams your chest! You can't breathe!</span>")
+	playsound(get_turf(A), 'sound/effects/hit_punch.ogg', 50, 1, -1)
+		D.losebreath = Clamp(D.losebreath + 2, 0, 10)
```
Comment (KayeArray) [inline hippiestation/code/datums/martial/krav_maga.dm]: umm this block is still indented, look you can see it is spaced differently to the rest

File: hippiestation/code/datums/martial/krav_maga.dm [hunk 8]
```diff
@@ -0,0 +1,19 @@
+/datum/martial_art/krav_maga/leg_sweep(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)
+	if(D.stat || D.IsKnockdown())
+		return FALSE
+	D.visible_message("<span class='warning'>[A] leg sweeps [D]!</span>", \
+					  	"<span class='userdanger'>[A] leg sweeps you!</span>")
+	playsound(get_turf(A), 'sound/effects/hit_kick.ogg', 50, 1, -1)
```
Comment (tortellinitony) [inline hippiestation/code/datums/martial/krav_maga.dm]: `playsound(A, 'sound/effects/hit_kick.ogg', 50, 1, -1)`

File: hippiestation/code/datums/martial/krav_maga.dm [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+	D.apply_damage(5, BRUTE)
+	D.Knockdown(20)
+	add_logs(A, D, "leg sweeped")
+return TRUE
+
+/datum/martial_art/krav_maga/quick_choke(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)//is actually lung punch
+	D.visible_message("<span class='warning'>[A] pounds [D] on the chest!</span>", \
+				  	"<span class='userdanger'>[A] slams your chest! You can't breathe!</span>")
+	playsound(get_turf(A), 'sound/effects/hit_punch.ogg', 50, 1, -1)
```
Comment (tortellinitony) [inline hippiestation/code/datums/martial/krav_maga.dm]: `playsound(A, 'sound/effects/hit_punch.ogg', 50, 1, -1)`

File: hippiestation/code/datums/martial/krav_maga.dm [hunk 10]
```diff
@@ -0,0 +1,19 @@
+/datum/martial_art/krav_maga/leg_sweep(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)
+	if(D.stat || D.IsKnockdown())
+		return FALSE
+	D.visible_message("<span class='warning'>[A] leg sweeps [D]!</span>", \
+					  	"<span class='userdanger'>[A] leg sweeps you!</span>")
+	playsound(get_turf(A), 'sound/effects/hit_kick.ogg', 50, 1, -1)
+	D.apply_damage(5, BRUTE)
+	D.Knockdown(20)
+	add_logs(A, D, "leg sweeped")
+return TRUE
```
Comment (tortellinitony) [inline hippiestation/code/datums/martial/krav_maga.dm]: Indent

File: hippiestation/code/datums/martial/krav_maga.dm [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+
+/datum/martial_art/krav_maga/quick_choke(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)//is actually lung punch
+	D.visible_message("<span class='warning'>[A] pounds [D] on the chest!</span>", \
+				  	"<span class='userdanger'>[A] slams your chest! You can't breathe!</span>")
+	playsound(get_turf(A), 'sound/effects/hit_punch.ogg', 50, 1, -1)
+	D.losebreath = Clamp(D.losebreath + 2, 0, 10)
+	D.adjustOxyLoss(5)
+	add_logs(A, D, "quickchoked")
+return TRUE
```
Comment (tortellinitony) [inline hippiestation/code/datums/martial/krav_maga.dm]: Indent

**Review by tortellinitony — DISMISSED** [review]

**Comment by PopNotes** [comment]
-1, krav magooba is fine as is

**Comment by tortellinitony** [comment]
yeah I don't like random nerf stuff like this

**Comment by Arzion5** [comment]
The nerfs were made because it is too lethal. The knockdown shouldn't be a free baton stun, and several lung punches can kill people because of the amount of oxyloss. As of currently, it is already more lethal than several available martial arts for traitors. I admit that what I did was copypaste into a new Hippie folder, and now I'm just worried about breaking anything.

**Comment by j-mie** [comment]
I'm not sure about this because we nerfed ranged weaponry for more dynamic fights to push for more CQC

File: hippiestation/code/datums/martial/krav_maga.dm [hunk 12]
```diff
@@ -3,7 +3,7 @@
 		return FALSE
 	D.visible_message("<span class='warning'>[A] leg sweeps [D]!</span>", \
 					  	"<span class='userdanger'>[A] leg sweeps you!</span>")
-	playsound(get_turf(A), 'sound/effects/hit_kick.ogg', 50, 1, -1)
+	playsound(A), 'sound/effects/hit_kick.ogg', 50, 1, -1)
```
Comment (ghost) [inline hippiestation/code/datums/martial/krav_maga.dm]: unbalanced brackets?

Comment (j-mie) [reply hippiestation/code/datums/martial/krav_maga.dm]: Looks like it, no compile error because he didn't include the file?!?

**Comment by Arzion5** [comment]
What about now?

**Comment by ghost** [comment]
you have to include this new file in hippiestation.dme by ticking the box beside it, otherwise the changes won't apply

**Comment by Arzion5** [comment]
Anything else I must do?

**Comment by ghost** [comment]
fix the compilation errors to pass jenkins

File: hippiestation/code/datums/martial/krav_maga.dm [hunk 13]
```diff
@@ -0,0 +1,19 @@
+/datum/martial_art/krav_maga/leg_sweep(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)
+	if(D.stat || D.IsKnockdown())
+		return FALSE
+	D.visible_message("<span class='warning'>[A] leg sweeps [D]!</span>", \
+					  	"<span class='userdanger'>[A] leg sweeps you!</span>")
+	playsound(A, 'sound/effects/hit_kick.ogg', 50, 1, -1)
+	D.apply_damage(5, BRUTE)
+	D.Knockdown(20)
+	add_logs(A, D, "leg sweeped")
+return TRUE
```
Comment (ghost) [inline hippiestation/code/datums/martial/krav_maga.dm]: indent this once

**Comment by ghost** [comment]
hurrah, it compiles

**Review by j-mie — APPROVED** [review]

**Pull request merged**